src/views/Home/components/EarnAPRCard.tsx

```diff
@@ -80,7 +84,10 @@ const EarnAPRCard = () => {
             {highestApr && !isFetchingFarmData ? (
               `${highestApr}%`
             ) : (
-              <Skeleton animation="pulse" variant="rect" height="44px" />
+              <>
+                <Skeleton animation="pulse" variant="rect" height="44px" />
+                <div ref={observerRef} />
```

Comment:
After first intersection, this div will not be rendered thus we will not get isIntersecting state updates therefore we will not make request over and over again